Packaging/Packaging.proj

```diff
@@ -0,0 +1,68 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="CreateStepPackage" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    
+    <!-- Change this when the runner goes up in version-->
+    <BootstrapperDirName>SonarQube.MSBuild.Runner-0.9</BootstrapperDirName>
+    
+    <!-- Needed to understand the output directory of referenced projects-->
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+
+    <TaskOutputDir>$(MSBuildThisFileFullPath)\..\..\VsoBuildSteps\$(Configuration)\</TaskOutputDir>
+    <TaskSourceDir>$(MSBuildThisFileFullPath)\..\..\VsoBuildStepSources\</TaskSourceDir>
+    <PreBuildTaskDestinationDir>$(TaskOutputDir)\SonarQubeInstallPreBuild\</PreBuildTaskDestinationDir>
+    <BootstrapperDirectory>$(PreBuildTaskDestinationDir)\$(BootstrapperDirName)</BootstrapperDirectory>
+    <TargetsFileSourcePath>$(MSBuildThisFileFullPath)\..\..\SonarQube.MSBuild.Tasks\bin\$(Configuration)\Targets\SonarQube.Integration.ImportBefore.targets</TargetsFileSourcePath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\SonarQube.Bootstrapper\SonarQube.Bootstrapper.csproj">
```

Comment:
The pre-build taks requires the boostraper so by adding a project reference to it we ensure it gets built first